frigate/events/maintainer.py

```diff
@@ -229,10 +230,10 @@ def handle_external_detection(self, type: str, event_data: Event) -> None:
                 Event.has_clip: event_data["has_clip"],
                 Event.has_snapshot: event_data["has_snapshot"],
                 Event.zones: [],
-                Event.data: { "type": "api" },
+                Event.data: {"type": event_data["type"]},
             }
             Event.insert(event).execute()
-        elif type == "end":
+        elif event_type == "end":
```

Comment:
you're broken all double-take installations now. at minimum

Reply:
in what way would this affect double-take at all?

Reply:
The code you highlighted is just changing the name of a local variable

Reply:
In no way. My mistake :(